auth/realm/base/src/main/java/org/wildfly/security/auth/realm/LegacyPropertiesSecurityRealm.java

```diff
@@ -419,6 +420,7 @@ public Builder setProviders(Supplier<Provider[]> providers) {
          * @return this {@link Builder}
          */
         public Builder setUsersStream(InputStream usersStream) {
+            assertNotNull(usersStream);
```

Comment:
Just a minor, for some of these, they actually can be set to null. If you look at the `usersStream` for example, you can see that there is a check on line 296 that differentiates whether this attribute is null or not.

Reply:
Good catch. Fixed it. Thanks.